lib/tasks/scripts/add_permission_to_access_level.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Creates the specified permission for users who have the specified access_level
+class AddPermissionToAccessLevel
+
+  attr_reader :permission_name, :access_level_name, :access_level, :permission
+
+  def initialize(permission_name, access_level_name)
+    @permission_name = permission_name
+    @access_level_name = access_level_name
+    @permission = Permissions::ALL_PERMISSIONS[permission_name]
+    @access_level = Permissions::ACCESS_LEVELS.select { |level| level[:name] == access_level_name }.first
+  end
+
+  def valid?
+    permission && valid_permission_for_access_level?
+  end
+
+  def create
+    users.each do |user|
+      permission_resources(user).each do |resource|
+        UserPermission
+          .where(user: user,
+                 permission_slug: permission[:slug],
+                 resource_type: resource[:resource_type],
+                 resource_id: resource[:resource_id])
+          .first_or_create
```

Comment:
> I realized that _technically_, this method will always only `create`, since the `user_is_a_match?` method will filter out users that already have the permission, but we should keep this in here to be safe, and safeguard against parallel processing.

That's not necessarily true. For example, a CVHO could have permissions of the same type for multiple facility groups.